app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -56,3 +53,55 @@ export const NOTES_CARD_DATA = [
     profilePicture: "https://i.pravatar.cc/300",
   },
 ];
+
+export const ASSIGNEES = [
+  {
+    label: "Oliver Smith",
+    value: "oliver",
+  },
+  {
+    label: "Eve Smith",
+    value: "eve",
+  },
+  {
+    label: "Sam Smith",
+    value: "sam",
+  },
+];
+
+export const TAGS = [
+  {
+    label: "High Priority",
+    value: "high",
+  },
+  {
+    label: "Low Priority",
+    value: "low",
+  },
+  {
+    label: "Deadline soon",
+    value: "deadline",
+  },
+];
+
+export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
+  title: yup.string().required("Title is required"),
+  description: yup.string().required("Description is required"),
+  assignee: yup
+    .object({
+      label: yup.string().required(),
+      value: yup.string().required(),
```

Comment:
You can use the [yup's oneOf validator](https://github.com/jquense/yup#schemaoneofarrayofvalues-arrayany-message-string--function-schema-alias-equals) and ensure label and value only takes values from your literal constant that is `ASSIGNEES`.